specs/src/specs/consensus.md

```diff
@@ -55,8 +55,9 @@
 | `MAX_VALIDATORS`                        | `uint16` | `64`         |         | Maximum number of active validators.                                                                                                                                |
 | `NAMESPACE_ID_BYTES`                    | `uint64` | `8`          | `byte`  | Size of namespace ID, in bytes.                                                                                                                                     |
 | `NAMESPACE_ID_MAX_RESERVED`             | `uint64` | `255`        |         | Value of maximum reserved namespace ID (inclusive). 1 byte worth of IDs.                                                                                            |
+| `SEQUENCE_BYTES`                        | `uint64` | `4`          | `byte`  | The number of bytes used to store the sequence length in the first share of a sequence                                                                              |
```

Comment:
is this a uint32? same question for the change below
```suggestion
| `SEQUENCE_BYTES`                        | `uint32` | `4`          | `byte`  | The number of bytes used to store the sequence length in the first share of a sequence                                                                              |
```

Reply:
I think the type for this constant only needs to contain the number of sequence bytes (`4`) so hypothetically this could be anything in the range from `uint8` to `uint64`. I'm not strongly opinionated on which type we use but want to point out that this type doesn't strictly need to be `uint32` even though the value stored in these four bytes will be ` uint32`. Does that make sense?  

On an related note, appconsts.go doesn't explicitly type integer constants so the specs diverge from the current implementation. https://github.com/celestiaorg/celestia-app/issues/1130